modules/gis/test/geojson-to-binary.spec.js

```diff
@@ -419,6 +422,18 @@ test('gis#geojson-to-binary position, featureId data types', async t => {
   t.end();
 });
 
+test('gis#geojson-to-binary with empty properties', async t => {
+  const response = await fetchFile(GEOJSON_NO_PROPERTIES);
+  const {features} = await response.json();
+  const {points, lines, polygons} = geojsonToBinary(features);
+
+  t.equal(points.properties[0].toString(), '[object Object]');
```

Comment:
Strange test using toString?

Reply:
The intention here is to check if properties is a plain object, instead of `undefined`, `null` or an array, for example, so I thought it was fair enough, but I can change it to
```js
foo instanceof Object && foo.length === undefined
```
if you prefer.

Reply:
I've finally changed it as `toString()` for a `null` value will generate an exception and the other check just returns `false`.